bin/README.md

```diff
@@ -53,6 +53,7 @@ on a remoate cluster when Hillview is installed
 
 * `delete-data.py`: delete a folder from all machines in a Hillview cluster
 * `deploy.py`: copy the Hillview binaries to all machines in a Hillview cluster
+* `deploy-hdfs.py`: download and setup HDFS to machines in a hillview cluster
```

Comment:
download and install HDFS on all ...

Reply:
Sorry the the grammar mistake. Updated

Reply:
One minor issue though. This does not deploy to all the machines if there are aggregators, only webserver and workers

Reply:
That's good